app/views/shared/partials/_footer-global.html.erb

```diff
@@ -0,0 +1,5 @@
+  <footer class="footer__global">
+    <small>An exploration by Viget’s intern class of 2017.
+  	&copy; Ground Rules 2017. All rights reserved.</small>
+    <a href="http://www.viget.com"><%= image_tag("vigetLogo.png") %></a>
```

Comment:
image tag needs `alt` text.

Also - cut the png this links to smaller.  You don't need an image as big as what you have.